src/workstation-gui/app/controllers/api/v1/cookbooks_controller.rb

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright:: Copyright Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+class Api::V1::CookbooksController < ApplicationController
+  # before_action :validate_creds, only: %i[login]
+  # skip_before_action :authenticate_api_requests!, only: %i[login]
+
+  def cookbooks
+    cookbooks_list = get_repository_cookbooks
+    result = []
+    cookbooks_list.each do |list|
+      result << list["cookbooks"]
+    end
+    result = result_post_pagination( result.flatten, params[:limit], params[:page])
+    render json: { cookbooks: result, message: "success", code: "200" }, status: 200
```

Comment:
Is it okay to have `success` as Constant and use it from there. So we have a single source. I will also do the same in this [PR](https://github.com/chef/chef-workstation/pull/2926/files)

Reply:
Once you do it and we merge, it i can change it ( i am waiting for merge) as it will let me add services and use some of yours model (for e.g if you made service folder and i did that too that will be duplication and conflict, so some chnages will be optimized after merge)